eng/versioning/version_client.txt

```diff
@@ -55,7 +55,7 @@ com.microsoft.azure:azure-servicebus-jms-spring-boot-starter;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot-metrics-starter;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot-tests;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot-test-core;2.3.2;2.3.3-beta.1
-
+com.microsoft.azure:spring-data-cosmosdb;2.3.0;2.3.1-beta.1
```

Comment:
@kushagraThapar, as per our discussion earlier we'll let this get checked in like this but part of the v4 updates will be to rename the groupId and change the artifactId to match the naming convention. Also, the work item you're creating will also encompass updating the versions to match this update to v4 cosmos which will update the major to 3.

Reply:
@JimSuplizio - Sounds good. Here is the issue : https://github.com/Azure/azure-sdk-for-java/issues/12490